src/commands/projects/list.ts

```diff
@@ -58,35 +96,81 @@ Use this command to list projects that are part of a particular organization.`
 
   async buildRequestParameters(options: Partial<flags.Output>): Promise<ProjectsListParams> {
     debug('buildRequestParameters()')
-    const {organization: orgId} = options
+    const {
+      'exclude-channels': excludeChannels,
+      'include-features': includeFeatures,
+      limit = defaultLimit,
+      offset,
+      organization: orgId,
+      sort: sortBy,
+      'with-name': filter,
+    } = options
 
-    return {orgId}
+    return {
+      excludeChannels,
+      includeFeatures,
+      ...(typeof filter === 'undefined' ? {} : {filter}),
+      ...(typeof limit === 'undefined' ? {} : {limit}),
+      ...(typeof offset === 'undefined' ? {} : {offset}),
+      ...(typeof orgId === 'undefined' ? {} : {orgId}),
+      ...(typeof sortBy === 'undefined' ? {} : {sortBy}),
+    }
   }
 
   doRequest(client: MixClient, params: ProjectsListParams): Promise<MixResponse> {
     debug('doRequest()')
     return ProjectsAPI.listProjects(client, params)
   }
 
-  outputHumanReadable(transformedData: any) {
+  outputHumanReadable(transformedData: any, options: any) {
     debug('outputHumanReadable()')
+    const {channelsColumn, columns, context, featuresColumn, timeColumns} = this
+    const count: number = context.get('count')
+    const totalSize: number = context.get('totalSize')
+    let isExcludeChannel = false
+    let isIncludeFeatures = false
+
     if (transformedData.length === 0) {
       this.log('No projects found.')
 
       return
     }
 
-    this.outputCLITable(transformedData, this.columns)
+    options['exclude-channels'] ? isExcludeChannel = true : isExcludeChannel = false
+    options['include-features'] ? isIncludeFeatures = true : isIncludeFeatures = false
+
+    let tableColumns = {}
+
+    isIncludeFeatures ? (isExcludeChannel ? tableColumns = {...columns, ...featuresColumn, ...timeColumns} :
+      tableColumns = {...columns, ...featuresColumn, ...channelsColumn, ...timeColumns}) :
+      (isExcludeChannel ? tableColumns = {...columns, ...timeColumns} :
+        tableColumns = {...columns, ...channelsColumn, ...timeColumns})
+
+    const resultInformation = count > 1 ? `1-${count}` : count
+
+    this.log(`\nShowing result${s(count)} ${chalk.cyan(resultInformation)} of ${chalk.cyan(totalSize)}.\n`)
+    this.outputCLITable(transformedData, tableColumns)
+
+    if (isIncludeFeatures) {
+      this.log('\nRun the command again with the --json flag to see all engine pack features.')
```

Comment:
The codebase does contain a number of message strings with embedded "\n" but to improve readability, let's replace those with standalone `this.log()`